js/models/languagesMd.js

```diff
@@ -717,7 +718,8 @@ module.exports = Backbone.Model.extend({
           socketError: "La URL del WebSocket fall&oacute;. Conectando a socket con la direcci&oacute;n predeterminada de ws://localhost:18466",
           contractError: "Este artículo no puede ser comprado" ,
           sellerError: "El servidor del vendedor ha rechazado la solicitud de compra",
-          checkPurchaseData: "Check your purchase data, such as quantity and Bitcoin refund address, to make sure it is valid" //not translated
+          checkPurchaseData: "Check your purchase data, such as quantity and Bitcoin refund address, to make sure it is valid", //not translated
+          storeUnavailable: "This store is currently unavailable."  //notTranslated
```

Comment:
Let's try to avoid using "store" in the language here since not every node is a store. Replace "store" with "page" and it should be good.